homeassistant/components/seventeentrack/__init__.py

```diff
@@ -1 +1,68 @@
 """The seventeentrack component."""
+from py17track import Client as SeventeenTrackClient
+from py17track.errors import SeventeenTrackError
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_SHOW_ARCHIVED, CONF_SHOW_DELIVERED, DOMAIN
+
+PLATFORMS = [Platform.SENSOR]
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): str,
+                vol.Required(CONF_PASSWORD): str,
+                vol.Optional(CONF_SHOW_ARCHIVED, default=False): bool,
+                vol.Optional(CONF_SHOW_DELIVERED, default=False): bool,
+            },
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
```

Comment:
We should keep this import in the `sensor.py`

Reply:
I'm not sure what you mean?
If I move this to `sensor.py` I get the following error:
`[ERROR] [CONFIG_SCHEMA] Integrations which implement 'async_setup' or 'setup' must define either 'CONFIG_SCHEMA', 'PLATFORM_SCHEMA' or 'PLATFORM_SCHEMA_BASE'`

Reply:
Currently the integration is setup in sensor.py (with I think either (async)setup(platform))

Reply:
removed setup from `sensor.py`